staging/src/k8s.io/apimachinery/pkg/watch/mux.go

```diff
@@ -96,10 +94,15 @@ func (obj functionFakeRuntimeObject) DeepCopyObject() runtime.Object {
 // The purpose of this terrible hack is so that watchers added after an event
 // won't ever see that event, and will always see any event after they are
 // added.
```

Comment:
update the comment on the `Watch` and `WatchforPrefix`, to indicate that it blocks until the watch is accepted.

Reply:
done.